src/Microsoft.AspNetCore.Mvc.Testing/breakingchanges.netcore.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.Testing.MvcWebApplicationBuilder<T0> where T0 : class",
```

Comment:
@javiercn was this a deliberate breaking change from 2.0?

Reply:
We didn't release this on 2.0. It's on the testing package. We added a baseline at the time to remove the warnings. (That's why I want a separate property for things that are not part of the release)

Reply:
Oh right!